src/test/java/teofilursan/tests/seleniumpart1/SeleniumExampleTest.java

```diff
@@ -1,9 +1,10 @@
 package teofilursan.tests.seleniumpart1;
 
+import ioanandroflorea.driver.BrowserManager;
```

Comment:
Please leave `import teofilursan.driver.BrowserManager;` and not add ioana's one as it's my package. Thanks!